### PR TITLE
Continue to improve docs organization

### PR DESCRIPTION
**Pull request by kathweinschenkprophecy**
to do: redirects

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1209052968527052

### CODE REVIEW

File: docs/concepts/fabrics/metadata-connections.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ tags:
   - data
 ---
 
-At Prophecy we're always looking to make your development faster. For teams that manage hundreds or thousands of tables, listing data catalogs, tables, and schemas can take precious minutes. In order to make this easier, Prophecy introduced Metadata Connections in release 3.2. The major benefit to Metadata Connections: [straightaway](/docs/metadata/metadata-connections.md) you can list and view your data directly in the [Prophecy UI](/docs/concepts/project/project.md#Environment-tab).
```
Comment (kathweinschenkprophecy) [inline docs/concepts/fabrics/metadata-connections.md:14]: should we rename this page? ask mei

Comment (kathweinschenkprophecy) [reply docs/concepts/fabrics/metadata-connections.md:14]: environment tab in projects = data warehouse metadata

**Review by atbida — APPROVED** [review]
lgtm!

File: docusaurus.config.js [hunk 2]
```diff
@@ -322,6 +370,9 @@ const config = {
           if (existingPath.includes("/copilot")) {
             return [existingPath.replace("/copilot", "/concepts/copilot")];
           }
+          if (existingPath.includes("/metadata/lineage")) {
+            return [existingPath.replace("/metadata/lineage", "/lineage")];
+          }
           return undefined;
```
Comment (atbida) [inline docusaurus.config.js:376]: image

Comment (atbida) [reply docusaurus.config.js:376]: image

Comment (atbida) [reply docusaurus.config.js:376]: image
****

**Pull request merged**